handlers/main.yml

```diff
@@ -20,3 +20,8 @@
   ansible.builtin.service:
     name: "{{ cvmfs_apache_service_name }}"
     state: reloaded
+
+- name: Restart gateway service
+  ansible.builtin.service:
+    name: cvmfs-gateway
```

Comment:
Other handlers use a variable to define the service name. This would be nice to keep the convention.